### PR TITLE
SLES 15 updates

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Implemented new install topic structure for EFM v4

### CODE REVIEW

**Comment by gsong** [comment]
⚠️ Rebased onto `develop`. Please update your local repo accordingly.

File: product_docs/docs/efm/3/03_installing_efm/03_efm3_rhel7_x86.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "RHEL 7/OL 7/CentOS 7 on x86_64"
+title: "RHEL 7 on x86_64"
```
Comment (nidhibhammar) [inline product_docs/docs/efm/3/03_installing_efm/03_efm3_rhel7_x86.mdx:2]: It should be RHEL 7/OL 7

File: product_docs/docs/pgpool/4.3/01_installing_and_configuring_the_pgpool-II.mdx [hunk 2]
```diff
@@ -402,7 +402,7 @@ After installing Pgpool-II on a Debian host, the configuration files are located
 
 <div id="using_the_SLES_host" class="registered_link"></div>
 
-## Installing Pgpool-II on a x86-64 SLES 12 Host
+## Installing Pgpool-II on aN SLES 12 x86-64 Host
```
Comment (nidhibhammar) [inline product_docs/docs/pgpool/4.3/01_installing_and_configuring_the_pgpool-II.mdx:405]: Looks like a typo - 'aN'

Comment (nidhibhammar) [reply product_docs/docs/pgpool/4.3/01_installing_and_configuring_the_pgpool-II.mdx]: Done the changes

**Pull request merged**